src/main/app/case/from-api-format.ts

```diff
@@ -21,6 +22,7 @@ console.log(checkboxConverter);
 const fields: FromApiConverters = {
   ...invert(formFieldsToCaseMapping),
   orderCollection: uploadedFilesFromApiApplicant1,
+  //hearingCollection: hearingCollectionFromApiApplicant1,
   // citizenUserDateOfBirth: data => ({
   //   citizenUserDateOfBirth: fromApiDate(data.citizenUserDateOfBirth),
   // }),
```

Comment:
remove commented out code if not required.

Reply:
removed the commented out code